snaps_k8s/provision/ansible_configuration.py

```diff
@@ -758,6 +767,13 @@ def __install_kubectl(k8s_conf):
 
     host_name, ip = config_utils.get_first_master_host(k8s_conf)
     ha_enabled = len(lb_ips) > 0
```

Comment:
Why is ha_enabled being set again? It was set above but never used.

Reply:
comment incorporated.  removed other ha_enabled flag. now this is the only flag that is being used.